docs/articles/documentation/recipes/basics/test-file-upload.md

```diff
@@ -0,0 +1,83 @@
+---
+layout: docs
+title: Test File Upload
+permalink: /documentation/recipes/basics/test-file-upload.html
+---
+# Test File Upload
+
+This recipe shows how to test your application's file upload functionality with TestCafe.
+
+[Full Example Code](https://github.com/DevExpress/testcafe-examples/tree/master/examples/upload-files)
+
+Assume the following HTML:
+
+```html
+<body>
+    <form action="http://localhost:3000/upload" method="post" enctype="multipart/form-data">
+        <input id="upload-input" type="file" name="files" multiple />
+        <input type="submit" id="upload-btn" value="Upload Files"/>
+    </form>
+</body>
+```
+
+This application utilizes the [JavaScript File API](https://developer.mozilla.org/en-US/docs/Web/API/File/Using_files_from_web_applications) for file upload. Users would usually perform the following actions to upload files:
```

Comment:
> This application utilizes the [JavaScript File API](https://developer.mozilla.org/en-US/docs/Web/API/File/Using_files_from_web_applications) for file upload.

Remove this sentence because application doesn't use such API.

Reply:
`Users would usually perform the following actions to upload files:`
->
`To upload files, the user need to perform the following steps:`

Reply:
edited